mandrill.go

```diff
@@ -321,6 +322,9 @@ func (c *Client) sendApiRequest(data interface{}, path string) (body []byte, err
 	if resp.StatusCode >= 400 {
 		resError := &Error{}
 		err = json.Unmarshal(body, resError)
+		if err != nil {
+			return nil, fmt.Errorf("HTTP %v: %v", resp.StatusCode, string(body))
+		}
 		return body, resError
```

Comment:
If the http response is not in JSON format, `resError` would be empty. In such case it's better to pass the status and message back to the caller.